xbmc/music/MusicDatabase.cpp

```diff
@@ -612,6 +623,12 @@ int CMusicDatabase::AddSong(const int idAlbum,
     if (m_pDS->num_rows() == 0)
     {
       m_pDS->close();
+      bool isBoxset = IsAlbumBoxset(idAlbum);
+      if (isBoxset && strDiscSubtitle.empty()) // boxset tag set but no disc subtitle in tags
+      { // As all discs in a boxset have to have a title, generate one in the form of 'Disc N'
```

Comment:
Move comment above bool declaration, again labelling section of code

Reply:
```
// As all discs in a boxset have to have a title, generate one in the form of 'Disc N
bool isBoxset = IsAlbumBoxset(idAlbum);
if (isBoxset && strDiscSubtitle.empty())
{ 
```

Reply:
`      // See if the current album is flagged as a boxset.  If it is, as boxset discs have to have a`
`      // title, if there is no title from the tags then generate our own in the form of 'Disc N'`
`      bool isBoxset = IsAlbumBoxset(idAlbum);`

Doesn't fit on here, but is that better ?  Before I commit it ?